bug-bounty.md

```diff
@@ -0,0 +1,46 @@
+# Bug Bounty
+
+The Moonbeam bug bounty program is focused on the Moonriver and Moonbeam Parachains (deployed to Kusama and Polkadot respectively) and dapps. It is focused on preventing:
+
+Moonbeam/Moonriver:
+* Thefts and freezing of principal of any amount
+* Thefts and freezing of unclaimed yield of any amount
+* Theft of governance funds 
+* Governance activity disruption
+* Network shutdown
+
+Website and Apps:
+* Website goes down
+* Leak of user data
+* Deletion of user data
+* Access to sensitive pages without authorization
+
+https://immunefi.com/bounty/moonbeam/
```

Comment:
Turns out this needs to be:
https://immunefi.com/bounty/moonbeamnetwork/

Reply:
Can you open another PR for that ? this one is merged already